### PR TITLE
added appveyor.yml

### PR DESCRIPTION
**Pull request by debris**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+environment:
+  matrix:
+  - FEATURES: ""
+
+platform:
+  - i686-pc-windows-msvc
```
Comment (ordian) [inline appveyor.yml]: cc #27

File: .travis.yml [hunk 2]
```diff
@@ -6,7 +6,8 @@ matrix:
   allow_failures:
     - rust: nightly
 script:
-  - cargo build
+  - cargo check --tests
```
Comment (dvdplm) [inline .travis.yml]: this will need `--all-features` I think?

File: uint/src/lib.rs [hunk 3]
```diff
@@ -9,7 +9,6 @@
 //! Efficient large, fixed-size big integers and hashes.
 
 #![cfg_attr(not(feature="std"), no_std)]
-#![cfg_attr(all(not(feature="std"), test), feature(alloc))]
```
Comment (dvdplm) [inline uint/src/lib.rs]: Can you motivate this change? I.e. I'm not sure I know why it was there in the first place, nor do I know what breaks removing it, but I do know substrate is using `uint` both with and without std.

File: fixed-hash/src/hash.rs [hunk 4]
```diff
@@ -677,11 +680,12 @@ mod tests {
 		assert!(r == u)
 	}
 
+	#[cfg(feature="uint_conversions")]
```
Comment (dvdplm) [inline fixed-hash/src/hash.rs]: +1

File: uint/src/lib.rs (hunk 3 above)
Comment (debris) [reply uint/src/lib.rs]: features are enabled only on the night channel, and this feature was not used anywhere :)

**Pull request merged**